libs/datasets/timeseries.py

```diff
@@ -778,6 +799,8 @@ def add_static_values(self, attributes_df: pd.DataFrame) -> "MultiRegionDataset"
         scalars_without_geodata = attributes_df.loc[
             :, attributes_df.columns.difference(GEO_DATA_COLUMNS)
         ]
+        # TODO(tom): When this assert always passes remove all the junk in _merge_attributes.
```

Comment:
This TODO is a bit confusing.  Should it live in `_merge_attributes` next to the code we want to remove?